udata/core/user/api.py

```diff
@@ -226,48 +228,62 @@ def get(self):
         search_parser.parse_args()
         return search.query(UserSearch, **multi_to_dict(request.args))
 
-    @api.secure
+    @api.secure('admin')
     @api.doc('create_user')
     @api.expect(user_fields)
     @api.marshal_with(user_fields, code=201)
     @api.response(400, 'Validation error')
     def post(self):
         '''Create a new object'''
-        form = api.validate(UserProfileForm)
+        form = api.validate(UserProfileAdminForm)
         user = form.save()
         return user, 201
 
 
+def is_not_available(user):
+    return user.active is False and (
+        current_user.is_anonymous or
```

Comment:
If the user is anonymous, it has not even the property sysadmin.

I could check the existence of the property, but it seems less readable to me.